Peer-to-peer Code review

razvanso opened the pull request.

razvanges committed:
Change positioning using floats

wuletawwonte committed:
Add tags hover effect

razvanges committed:
WIP: Add rounded corner effect on bg shapes

wuletawwonte committed:
Add background radius to rounded-corner container

razvanges committed:
Add rounded-corner to about section

wuletawwonte commented:
Hi @razvanso 
------
You have done a good job with the styling of the elements. I think it would be better if linter errors are fixed.  
### Required changes
Please fix linter errors because it is a good practice to fix linter errors and makes your code clean and readable.

razvanges committed:
Incorporate feedback

- bg image not centered properly (use cover instead)
- work-card images not centered

razvanges committed:
Fix linting errors
